package.json

```diff
@@ -46,5 +46,8 @@
     "nodeArguments": [
       "--loader=ts-node/esm"
     ]
+  },
+  "engines": {
+    "node": ">=16.14"
```

Comment:
`AbortSignal.timeout(delay)` was added in 16.14.0, hence the update here
see: https://nodejs.org/docs/latest-v16.x/api/globals.html#static-method-abortsignaltimeoutdelay

Reply:
Also noting that node v18 is not yet available as a managed lambda runtime, but that might land in the next few weeks https://github.com/aws/aws-lambda-base-images/issues/47#issuecomment-1292011841